test/about/ledgerPanelTest.js

```diff
@@ -138,6 +139,39 @@ describe('Regular payment panel tests', function () {
         .click(walletSwitch)
         .waitForEnabled(addFundsButton)
     })
+
+    it('clicks BAT TOS link when payments are enabled', function * () {
+      yield this.app.client
+        .tabByIndex(0)
+        .loadUrl(prefsUrl)
+        .waitForVisible(paymentsTab)
+        .click(paymentsTab)
+        .waitForVisible(paymentsWelcomePage)
+        .waitForVisible(walletSwitch)
+        .click(walletSwitch)
+        .waitForEnabled(addFundsButton, ledgerAPIWaitTimeout)
```

Comment:
I am not sure why this is needed but this line fixes the test failure.

Reply:
because we add timeout to the test sequence which is representing time spent to crate wallet